cdap-api/src/main/java/io/cdap/cdap/api/app/Application.java

```diff
@@ -32,4 +32,21 @@
    * @param context Used to access the environment, application configuration, and application (deployment) arguments
    */
   void configure(ApplicationConfigurer configurer, ApplicationContext<T> context);
+
+  /**
+   * Returns if application supports config update or not.
+   */
+  default boolean isConfigUpdateSupported() {
```

Comment:
Call it `isUpdateConfigSupported` to match with the `updateConfig` name.

Reply:
I changed the method to isUpdateSupported.